src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter_nodelts.py

```diff
@@ -25,16 +25,6 @@ def test_containerapp_session_code_interpreter_nodelts_e2e(self, resource_group)
         sessionpool_list = self.cmd("containerapp sessionpool list -g {}".format(resource_group)).get_output_in_json()
         self.assertTrue(len(sessionpool_list) == 0)
 
-        # Create PythonLTS SessionPool with default container type which is PythonLTS
-        sessionpool_name_pythonlts = self.create_random_name(prefix='sppythonlts', length=24)
-        self.cmd('containerapp sessionpool create -g {} -n {} --cooldown-period {}'.format(
```

Comment:
Because this file should be focus on NodeLTS session pool, and all the validation is targeting NodeLTS python pool rather than the Python session pool created here.
The tests and validations against PythonLTS session pools are duplicated in another test file ContainerAppSessionCodeInterperterTests.